src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -167,19 +167,27 @@ Do you want to start the scheduler service now? [Y/n]  y
 If a problem occurs, check output in /var/log/proactive/scheduler
 ....
 
+The installation script will create a symbolic link called `default` under the root path chosen for the installation that will point to the current version folder.
+
+Accordingly, the ProActive server installation can be accessed using `/opt/proactive/default` path.
+
 Once the server is up, ProActive Web Portals are available at `\http://localhost:8080/`. The user can access to any of the four portals using the default credentials: *admin/admin*
 
 Your ProActive Scheduler is ready to execute <<_glossary_job,*Jobs*>>!
 
 ==== How to upgrade ProActive on Linux
-Before you upgrade to a new version of ProActive (ex 10.0), you first need to back up the configuration and data that you want to restore (DB, workflows, etc) from the old ProActive installation (ex 8.4). Basically, you have to back up the content of the following folders:
 
-*  `$PROACTIVE_HOME/data` which contains all databases (scheduler, catalog, rm) and workflows of the different users.
-*  `$PROACTIVE_HOME/config` which contains configuration files required to setup the server initial settings, define the network properties, manage the authentication, internal accounts, access permissions, etc.
+First, https://www.activeeon.com/community-downloads[Download^] ProActive Scheduler and *unzip* the archive inside a different folder from the previous installation (for example if ProActive is installed in `/opt/proactive/`, download and unzip the new version in `/home/user`).
+
+To upgrade ProActive, the user has to run the `install.sh` script located under the `tools` folder of the extracted new version as *root* and follow the installations steps provided in an interactive mode.
```

Comment:
this is not correct.
when you unzip it there is folder activeeon_enterprise-pca_server-linux-x64-14.1.0-SNAPSHOT-<DATE>
both packaged and unzipped version is placed in the same place (in case of nebulous home/ubunutu/)

you do not access root and proactive user until the installation is done. then it is to set up config file of scheduler (any documentation regarding this? if yes would interlink it in installation process or atleast mention that additional setups can be done there (as there are explanations in config file)

Reply:
For me is also mandatory to mention that the previous installation should not be removed in case the user is to preserve the data.

Reply:
@ankicabarisic 

what is not correct ?

the installed version is put in /opt/proactive
the downloaded version should be put inside a different folder as I mentioned (for example /home/user)

the fact that you observe that there is already a downloaded version in /home/ubuntu in the nebulous server is irrelevant to this documentation. As far as I'm concerned you can remove previously downloaded versions in /home/ubuntu, but not previously installed versions in /opt/proactive. Hope that you understand the nuance.

Ok I can add a line mentioning that the previous installation should not be removed before the upgrade

Reply:
it is confusing as you are talking about installation and version  :)
so it is easily misunderstood.  i was thinking you are talking regarding the installation files, so was confused why they should be in /opt/proactive for previous one. 

so I guess to improve it, it is to differentiate between two and call them consistently with the same name.

Reply:
I don't know what you don't understand in my sentence

Reply:
I improved some phrasing and explanations, can you check if it's more understandable now ?

Reply:
@fviale For me this would be more clear (if is correct):
First, [download](https://www.activeeon.com/community-downloads) the ProActive Scheduler and unzip the archive into a new folder, separate from the previous installation (e.g., if ProActive was installed in /opt/proactive/, place the new version in /home/user). You can safely delete the folder of the previously extracted version, but keep the original installation folder to preserve your data.

To upgrade ProActive, run the install.sh script located in the tools folder of the new version’s directory as root. Follow the interactive installation steps, ensuring that you install it to the same directory as the previous installation (e.g., /opt/proactive/).

Reply:
I think you didn't read my updates in the code, don't refer to what you see above this discussion as it is out-of-date.

Open the "Files changed" to see the latest version

Reply:
hello, you made an update just when I wrote the above. 
yes, I find it much clearer now.

the only thing to reconsider is if it is to request them to run it as the root user.

Reply:
It is written here:

`To upgrade ProActive, the user has to run the `install.sh` script located under the `tools` folder of the extracted new version as *root* or *sudo* and follow the installations steps provided in an interactive mode.`

Of course the installation has to be run as root